agent/rfswarm-base-image/Dockerfile

```diff
@@ -5,7 +5,8 @@ RUN   pip install rfswarm-agent
 RUN   pip install robotframework-requests 
 RUN   pip install robotframework-datadriver
 
-COPY --chmod=+x agent/rfswarm-base-image/entrypoint.sh /opt/bin/entrypoint.sh
+COPY  agent/rfswarm-base-image/entrypoint.sh /opt/bin/entrypoint.sh
```

Comment:
I see what you had done with the `--chmod=+x`and that "should" have worked, but then again in the documentation for COPY (https://docs.docker.com/reference/dockerfile/#copy---chown---chmod) it gives many examples for chown, and the documentaton seems to focus exclusively on this option and only one example for chmod (`--chmod=644`), so perhaps this is a bug in docker? This change should be more reliable :+1: